lib/rialto/etl/writers/sparql_writer.rb

```diff
@@ -102,11 +102,13 @@ def send_batch(batch)
         # Post the statements to the SPARQL endpoint
         # @param [String] SPARQL statements to send
         def post(statements)
-          client.post do |req|
+          response = client.post do |req|
             req.body = statements
           end
+          raise "#{response.reason_phrase}: #{response.status}  (#{response.body})" unless response.success?
```

Comment:
can you add a test for this behavior?

Reply:
Or better yet, can I write any tests for Sparql Writer ...

Reply:
@jcoyne Test added.

Reply:
For some reason SparqlWriter#close() is not getting called in our test suite (which is what cause the coverage to fall).  Is that expected?

Reply:
Added a test for close.